include/egg/math/eggMatrix.h

```diff
@@ -62,7 +62,7 @@ struct Matrix34f {
     void transposeTo(Matrix34f &to) const;
     void copyFrom(const Matrix34f &other) {
         for (int i = 0; i < 12; i++) {
-            (*this)(i) = other(i);
+            arr[i] = other.arr[i];
```

Comment:
This function was confirmed to use the () operator
```
19] copyFrom__Q23EGG9Matrix34fFRCQ23EGG9Matrix34f (func,weak) found in eggND.a 
       20] __cl__Q23EGG9Matrix34fCFi (func,weak) found in eggND.a
```

Reply:
Will change it back, but there is one case in `dShpProc1_c::draw` where the existing code caused regswaps and the explicit indexing worked. So maybe there are even more inlines... https://decomp.me/scratch/xwU5p

Reply:
https://decomp.me/scratch/QEf2I A long shot, but maybe?? I did not check how it effects other things. 
There is also the chance of them just doing the loops locally.

Reply:
Regresses dAcOPoolCock_c::actorExecute, which does matrix copies via GPRs :/